scripts/docker/enable-winrm.ps1

```diff
@@ -15,7 +15,11 @@ if (Test-Path A:\install-containers-feature.ps1) {
 Stop-Service winrm
 . sc.exe config winrm start= delayed-auto
```

Comment:
The WinRM service is set to delayed-auto as I also install the Containers feature and optionally the Hyper-V feature. It really needs a reboot that does another reboot after some seconds.
I had problems while WinRM is set to auto instead of delayed-auto. The other reboot is done before the 120 second delay, so WinRM port is still not listening and packer can't connect.
Not the best provision script as it mixes the name "Enable WinRM" with installing Containers feature, but that was the easiest place to put it in and save one other reboot.